dashboard/app/assets/stylesheets/application.scss

```diff
@@ -983,7 +983,7 @@ $default-modal-width: 640px;
 .header_right {
   position: absolute;
   float: left;
-  right: 25px;
+  right: 16px;
```

Comment:
Great catch.  I'd discovered this when messing with the DOM and then forgot when implementing here.